src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/SetAzureWebApp.cs

```diff
@@ -151,7 +154,8 @@ public override void ExecuteCmdlet()
                             Location = location,
                             ServerFarmId = WebApp.ServerFarmId,
                             Identity = parameters.Contains("AssignIdentity") && AssignIdentity ? new ManagedServiceIdentity("SystemAssigned", null, null) : WebApp.Identity,
-                            HttpsOnly = parameters.Contains("HttpsOnly") ? HttpsOnly : WebApp.HttpsOnly
+                            HttpsOnly = parameters.Contains("HttpsOnly") ? HttpsOnly : WebApp.HttpsOnly,
+                            ClientAffinityEnabled = parameters.Contains("ClientAffinityEnabled") ? ClientAffinityEnabled : WebApp.ClientAffinityEnabled
```

Comment:
@jocawtho in the scenario test above, you get the value for `ClientAffinityEnabled` from a web app, but if you hadn't provided a value at all, this line would still get the value from the web app. Is there a scenario where the user would provide the value for `ClientAffinityEnabled` from anywhere but a web app? If not, then we should remove the parameter and just set the value to `WebApp.ClientAffinityEnabled` all the time

Reply:
Customer's should be able to set this setting without getting the setting from a webApp, but I chose to get the bool value from a webApp property to ensure the test's accuracy by setting the opposite value no matter what the default may be.